### DISCUSSION TITLE
ESP3D via Ethernet?

### DISCUSSION THREAD

**Discussion by rubin110** [luc-github/ESP3D#1036] [Questions]
I would like to get rid of my Rpi running Octoprint and switch over to ESP3D talking to Octoprint elsewhere on my network. I have a hard time trusting wifi for communication, so I want to use an ESP32 with an Ethernet port. I've been searching and searching but haven't found a clear answer as to how this can be achieved with ESP3D. I say that an Olimex board is supported but nothing beyond Ethernet configuration and what pins to use. Before I dig in more I figured asking here wouldn't hurt. 

Any guidance would be most appreciated. Thanks!

**Comment by luc-github** [comment]
Ethernet is supported in esp3d 3.0 
tested board are listed here https://esp3d.io/esp3d/v3.x/hardware/esp_boards/index.html

**Comment by rubin110** [comment]
Thank you for the quick response. Is Ethernet configured within the UI at this point?

**Reply by luc-github** [reply]
no you need to configure it , there is a configurator to help https://luc-github.github.io

**Comment by rubin110** [comment]
Ah ok. Thanks for the direction!
